build.gradle

```diff
@@ -228,6 +228,26 @@ project('spring-rabbit') {
 
 }
 
+project('spring-rabbit-test') {
+	description = 'Spring Rabbit Test Support'
+	dependencies {
+
+		compile project(":spring-rabbit")
+		compile ("junit:junit:$junitVersion") {
+			exclude group: 'org.hamcrest'
+		}
+		compile "org.hamcrest:hamcrest-all:$hamcrestVersion"
+		compile ("org.mockito:mockito-core:$mockitoVersion") {
+			exclude group: 'org.hamcrest'
+		}
+		testCompile project(":spring-rabbit").sourceSets.test.output
```

Comment:
So, you decided do not copy/paste `BrokerRunning` and therefore do not expose it as a high-level testing util.